.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
```

Comment:
actually we will need to link to internal JIRA. Otherwise, we lost most context on many PRs

Reply:
Perhaps just the name of the internal JIRA is sufficient.  Not an actual hyperlink.

Reply:
sure...although I am not clear if it makes any difference to users not on VPN

Reply:
I agree with Gilbert.  No hyperlinks to anything internal.  
You may mention a JIRA number (at most), but no hyperlinks.

Reply:
I've addressed this elsewhere.